push:
  num_points: 2
  inter_points:
    - [2.0, -0.3, 1.0, 0.65, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0]
    - [2.0, 0.30, 1.0, 0.65, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0]
  object:
    position: [0.0, 1.0, 0.6]
    quaternion: [1.0, 0.0, 0.0, 0.0]
  start_pt: [-2.0, 0.0, 1.2]
  end_pt: [2.0, 0.0, 1.2]

pull:
  num_points: 4
  inter_points:
    - [2.0, -0.4, 0.20, 0.98, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0]
    - [2.0, -0.2, 0.20, 0.73, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    - [2.0, 0.20, 0.20, 0.73, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    - [2.0, 0.2, -0.20, 0.73, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    - [2.0, -0.2, -0.2, 0.73, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
  object:
    position: [0.0, 0.0, 0.63]
    quaternion: [1.0, 0.0, 0.0, 0.0]
  start_pt: [-3.0, 0.0, 1.2]
  end_pt: [3.0, 0.0, 1.2]
    
lift:
  num_points: 1
  inter_points:
    - [2.0, 0.0, 0.0, 0.75, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0]
  object: 
    position: [0.0, 0.0, 0.6]
    quaternion: [1.0, 0.0, 0.0, 0.0]
  start_pt: [-3.0, 0.0, 1.2]
  end_pt: [1.0, 0.0, 1.5]

press:
  num_points: 1
  inter_points:
    - [2.0, 0.0, 0.0, 0.60, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0]
  object: 
    position: [0.0, 0.0, 0.6]
    quaternion: [1.0, 0.0, 0.0, 0.0]
  start_pt: [-2.0, 0.0, 1.2]
  end_pt: [2.0, 0.0, 1.2]

write:
  num_points: 6
  inter_points:
    - [2.0, -0.2, 0.180, 0.66, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0]
    - [2.0, 0.20, 0.180, 0.66, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 1.0, 0.0, 1.0]
    - [2.0, 0.20, -0.16, 0.66, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 0.0, 1.0, 1.0]
    - [2.0, 0.02, -0.16, 0.66, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 1.0, 0.0, 1.0]
    - [2.0, 0.02, 0.180, 0.66, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    - [2.0, -0.2, -0.20, 0.66, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  object: 
    position: [0.0, 0.0, 0.62]
    quaternion: [1.0, 0.0, 0.0, 0.0]
  start_pt: [-3.0, 0.0, 1.5]
  end_pt: [3.0, 0.0, 1.5]